### PR TITLE
fix(build): remove `check_required_components` for meson build

### LINKED ISSUE #1568: [1.9.6 regression] 'find_package(jsoncpp REQUIRED)' fails: Unknown CMake command "check_required_components"
**Describe the bug**
```
$ cat CMakeLists.txt 
find_package(jsoncpp REQUIRED)
```

```
CMake Error at /usr/local/lib/cmake/jsoncpp/jsoncppConfig.cmake:31 (check_required_components):
  Unknown CMake command "check_required_components".
Call Stack (most recent call first):
  CMakeLists.txt:1 (find_package)
```

**Desktop (please complete the following information):**
 - OS: FreeBSD 14.1
 - cmake-3.29.6
 - Meson version: 1.5.1

### PR DESCRIPTION
**Pull request by chenrui333**
fixes #1568

relates to https://github.com/Homebrew/homebrew-core/pull/189352

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -179,8 +179,8 @@ if(JSONCPP_WITH_PKGCONFIG_SUPPORT)
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 endif()
 
+include(CMakePackageConfigHelpers)
```
Comment (baylesj) [inline CMakeLists.txt:182]: Why is this change necessary?

Comment (chenrui333) [reply CMakeLists.txt:182]: yeah, that is not needed actually, removing it
